src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -94,70 +90,45 @@ const defaultProps = {
     isThreadParentMessage: false,
     isApprovedOrSubmittedReportAction: false,
     isFragmentContainingDisplayName: false,
-    displayAsGroup: false,
```

Comment:
Did we need to remove this default? It makes sense that if we do not pass the prop that it should fallback to `false`? 🤔

Reply:
I'm just not a fan of default props unless they provide apparent benefits. This component doesn't have many usages (two, if I'm not mistaken), so why do we need to split the behavior into "standard" and "falling back to default"?

But, specifically, this place is a result of conflict resolution; I added this prop independently from some other PR, and we took different strategies for handling defaults.

Reply:
Ok, well, I think _we_ are "fans" of it because we do it all over the place. 

If there is a default then you don't need to explicitly pass a prop, less changes to review, etc. 

Is there a style guide rule that says we should always be explicit about this? And if not, can you accept the feedback?

Reply:
I accepted your feedback; just didn't agree with it.

I don't think this is worth delaying this PR, though. Changed.